selectors/onboarding.ts

```diff
@@ -95,24 +95,6 @@ const getSteps = createSelector(
     },
 );
 
-// show Onboarding task list if they haven't skipped/finished the legacy steps or there are legacy steps unfinished
-export const showOnboardingTaskListToExistingUsers = createSelector(
```

Comment:
this is no longer needed, since what is actually important are the SKIP and HIDE preferences. And the new function `hasLegacyNextStepsPreferences` takes care of validating if any of the steps exists, meaning that is an existing customer that was already interacting with the legacy nextSteps, but what determines if the new onboarding is shown or not are  HIDE=false and non-existance of SKIP nor SKIP=false